wigwam/_docker_cmake.py

```diff
@@ -86,3 +86,43 @@ def cmake_build_dockerfile(base: str) -> str:
     dockerfile += "RUN cmake --build $BUILD_PREFIX --parallel"
 
     return dockerfile
+
+
+def cmake_install_dockerfile(base: str, ld_lib: str) -> str:
+    """
+    Creates a dockerfile for installing with CMAKE.
+
+    Parameters
+    ----------
+    base : str
+        The base image tag.
+    ld_lib : str
+        The name of the linking library file (e.g. "lib" or "lib64".)
```

Comment:
Let's rename this parameter to `libdir`.

```suggestion
    libdir : str
        The base name of the directory where object libraries are stored
        (e.g. "lib" or "lib64"). This should match the value of `LIBDIR`
        in CMake's `GNUInstallDirs` module 
        (https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html).
```